packages/terra-functional-testing/src/config/utils/getConfigurationOptions.js

```diff
@@ -29,8 +31,8 @@ const getConfigurationOptions = (options) => {
   return {
     baseUrl: `http://${externalHost || getIpAddress()}:${externalPort || 8080}`,
     capabilities: getCapabilities(browsers, !!gridUrl),
-    hostname: gridUrl || (useSeleniumStandaloneService ? 'standalone-chrome' : 'localhost'),
-    port: gridUrl ? 80 : 4444,
+    hostname: dockerUrl || gridUrl || (useSeleniumStandaloneService ? 'standalone-chrome' : 'localhost'),
+    port: dockerPort || (gridUrl ? 80 : 4444),
```

Comment:
Adding a `port` param is reasonable, but i'm not sure what the difference is between the `dockerUrl` and `gridUrl`.

```suggestion
    hostname: gridUrl || (useSeleniumStandaloneService ? 'standalone-chrome' : 'localhost'),
    port: port || (gridUrl ? 80 : 4444),
```

Reply:
I tried using gridUrl. There's logic elsewhere in the package tied to gridUrl to set http protocol and perform status checks etc, which prevents me from using it.